ecosystem/lava-sdk/src/stateTracker/stateQuery/state_chain_query.ts

```diff
@@ -120,14 +125,50 @@ export class StateChainQuery {
           continue;
         }
 
+        downtimeParams = pairingResponse.getDowntimeParams();
+
         const providers = pairing.getProvidersList();
         timeLeftToNextPairing = pairing.getTimeLeftToNextPairing();
-        const currentEpoch = pairingResponse.getPairing()?.getCurrentEpoch();
+        currentEpoch = pairingResponse.getPairing()?.getCurrentEpoch();
         if (!currentEpoch) {
           throw Logger.fatal(
             "Failed fetching current epoch from pairing request."
           );
         }
+
+        // check if epoch has not changed
+        if (
+          this.latestEpoch == currentEpoch &&
+          virtualEpoch == 0 &&
+          downtimeParams != undefined
+        ) {
+          const lastBlockTime = pairingResponse.getLatestBlockTime();
+          const downtimeDuration = downtimeParams.getDowntimeDuration();
+          const epochDuration = downtimeParams.getEpochDuration();
+
+          if (
+            lastBlockTime != undefined &&
+            downtimeDuration != undefined &&
+            epochDuration != undefined
+          ) {
+            const delay = Date.now() - lastBlockTime;
```

Comment:
we should not be using this to calculate the delay,
1. you treat all blocks the same no matter if it's a block in the beginning of epoch or the end of an epoch but the delay calculation is completely different
2. we do not have access to the query if the node is shut down
3. the time of the block or the current time will not be that different, and it's not a good enouch reason to add it to the sdk query

i want us to remove the sdk query lastBlockTime, we can take the delay to be the epoch change time that the sdk knows of (the time it identifies a change in the epoch) or the start time, like we do in chain_tracker.go